core/fortran-example/tutorial/04_aosoa/main.cpp

```diff
@@ -0,0 +1,119 @@
+/****************************************************************************
+ * Copyright (c) 2018-2019 by the Cabana authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the Cabana library. Cabana is distributed under a   *
+ * BSD 3-clause license. For the licensing terms see the LICENSE file in    *
+ * the top-level directory.                                                 *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#include <Cabana_Core.hpp>
+
+#include <iostream>
+
+#include "veclen.h"
+
+//---------------------------------------------------------------------------//
+// AoSoA example.
+//---------------------------------------------------------------------------//
+/*
+  Cabana array-of-structs-of-arrays (AoSoAs) is simply a list of Cabana
```

Comment:
Instead of copy-pasting these comments from the C++ tutorial let's update them to reflect what this Fortran example is doing.

Reply:
Done.